myosuite/envs/env_base.py

```diff
@@ -486,6 +487,7 @@ def seed(self, seed=None):
         """
         self.input_seed = seed
         self.np_random, seed = gym.utils.seeding.np_random(seed)
+        self.np_random = NPRandomVersionWrapper(self.np_random)
```

Comment:
> self.np_random, seed = gym.utils.seeding.np_random(seed)
> self.np_random = NPRandomVersionWrapper(self.np_random)

1. Seems like we are initializing the np_random twice. 
2. What's this new NPRandomVersionWrapper? Why is this needed now? Did something change?

Reply:
Needed to support multiple version of numpy random integers https://github.com/MyoHub/myosuite/pull/223
cc: @P-Schumacher

Reply:
it's a wrapper around np_random that allows the use of older numpy generators which do not have `intergers()` but rely on `randint()`

this should be good to go, or not?

Reply:
Seems like we are initializing the np_random twice in consecutive line. One of hem should be nuked.

Reply:
it's a wrapper around it, it's not initialized twice. i will integrate the gym seeding line into the wrapper, if that makes the code clearer

Reply:
done